specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2697,6 +3231,230 @@
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RedisCacheAccessPolicy",
+          "description": "Base properties required to create or update an access policy."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "Parameters required for creating an access policy for the redis cache",
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyProperties": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of access policy",
+          "readOnly": true,
+          "enum": [
+            "Updating",
+            "Succeeded",
+            "Deleting",
+            "Deleted",
+            "Canceled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "AccessPolicyProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "type": {
+          "type": "string",
+          "description": "Built-In or Custom access policy",
+          "readOnly": true,
+          "enum": [
+            "Custom",
+            "BuiltIn"
+          ],
+          "x-ms-enum": {
+            "name": "AccessPolicyType",
+            "modelAsString": true
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/RedisCacheAccessPolicy"
+        }
+      ],
+      "description": "All properties of an access policy.",
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyResponse": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RedisCacheAccessPolicyProperties",
+          "description": "Properties of an access policy."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Response to get/put access policy.",
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyListResponse": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RedisCacheAccessPolicyResponse"
+          },
+          "description": "List of access policies (with properties) of a Redis cache."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link for next set."
+        }
+      },
+      "description": "List of access policies (with properties) of a Redis cache.",
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyAssignment": {
+      "description": "Properties for an access policy assignment",
+      "properties": {
+        "objectId": {
+          "type": "string",
+          "description": "Object Id to assign access policy to"
+        },
+        "name": {
+          "type": "string",
+          "description": "User name"
```

Comment:
>"description": "User name" [](http://example.com/codeflow?start=10&length=26)

I think the description needs to explain more context that this is a username in the redis dataplane.

Reply:
Updated description to "User friendly name for object id. Also represents username for token based authentication"